replica/table.cc

```diff
@@ -933,7 +940,7 @@ compaction_group& tablet_storage_group_manager::compaction_group_for_sstable(con
                                           sst->get_filename(), first_id, last_id));
     }
 
-    auto& sg = storage_groups()[first_id];
+    auto& sg = storage_groups().at(first_id);
```

Comment:
ditto, check for tablet existence and abort with internal_error if it's missing instead of terminating due to throwing in noexcept context.